test/integration/http2_integration_test.cc

```diff
@@ -195,6 +195,110 @@ TEST_P(Http2IntegrationTest, Trailers) { testTrailers(1024, 2048); }
 
 TEST_P(Http2IntegrationTest, TrailersGiantBody) { testTrailers(1024 * 1024, 1024 * 1024); }
 
+TEST_P(Http2IntegrationTest, IdleTimoutBasic) { testIdleTimeoutBasic(); }
+
+TEST_P(Http2IntegrationTest, IdleTimeoutWithTwoRequests) { testIdleTimeoutWithTwoRequests(); }
+
+TEST_P(Http2IntegrationTest, IdleTimeoutWithSimultaneousRequests) {
+  simultaneousRequestsWithIdleTimeout();
+}
+
+// Interleave two requests and responses and make sure that idle timeout is handled correctly.
+void Http2IntegrationTest::simultaneousRequestsWithIdleTimeout() {
```

Comment:
No need for separate function that is the only thing called by a test. Just inline this entire thing into the single test that is calling it.

The reason some of the other tests above call testTrailers() is because they're calling the same function with different parameters in each case.